designsafe/static/styles/app-version-list.css

```
@@ -5,6 +5,9 @@
     background-color: #F4F4F4;
     padding: 20px;
 }
+.s-app-version-list:not(:has(article)) {
+    text-align: center;
+}
```

Should I instead use a `.s-app-version-list--single` or something? E.g.

```suggestion
.s-app-version-list--single {
    text-align: center;
}
```

The `s-` (scope) patterns are intend to serve as a scope/container class for markup or third-party classes. I've never really found a reason to define when to use modifier class (`…--modifier`) them.